<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TCP/UDP </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QLabel" name="text_openfile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>210</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Open the file you wish to send.</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="sendreceive">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="sender_ui">
     <widget class="QTextBrowser" name="text_show">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>480</width>
        <height>321</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>238</red>
            <green>238</green>
            <blue>236</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>238</red>
            <green>238</green>
            <blue>236</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>239</red>
            <green>239</green>
            <blue>239</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graph_show">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>480</width>
        <height>321</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>321</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>238</red>
            <green>238</green>
            <blue>236</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>238</red>
            <green>238</green>
            <blue>236</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>239</red>
            <green>239</green>
            <blue>239</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
     </widget>
     <widget class="QFrame" name="gridFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>471</width>
        <height>161</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QLabel" name="text_yourIP">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>220</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
      <widget class="QLabel" name="text_protocol">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>13</y>
         <width>117</width>
         <height>17</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Sender protocol: </string>
       </property>
      </widget>
      <widget class="QLabel" name="text_toport">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>220</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QComboBox" name="sel_protocol">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>59</width>
         <height>25</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>TCP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UDP</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="but_send">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Set Up</string>
       </property>
      </widget>
      <widget class="QLabel" name="text_sent">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>130</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="send">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="text_filename">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>300</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>You are reading &quot;&quot; </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="receive_ui">
     <widget class="QLabel" name="text_filename_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>300</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>You are reading &quot;&quot; </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="text_show_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>480</width>
        <height>321</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>321</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>238</red>
            <green>238</green>
            <blue>236</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>238</red>
            <green>238</green>
            <blue>236</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>239</red>
            <green>239</green>
            <blue>239</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graph_show_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>480</width>
        <height>321</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>321</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>238</red>
            <green>238</green>
            <blue>236</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>238</red>
            <green>238</green>
            <blue>236</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>239</red>
            <green>239</green>
            <blue>239</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
     </widget>
     <widget class="QFrame" name="gridFrame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>471</width>
        <height>161</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QLabel" name="text_clientIP">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>220</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Connection IP:  </string>
       </property>
      </widget>
      <widget class="QLabel" name="text_client">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>13</y>
         <width>161</width>
         <height>17</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Client properties:</string>
       </property>
      </widget>
      <widget class="QLabel" name="text_clientport">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Receiving through port:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="but_connect">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="fiel_clientport">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>90</y>
         <width>141</width>
         <height>25</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
       <property name="placeholderText">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QComboBox" name="fiel_clientIP">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>251</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="text_status">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>130</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <zorder>graph_show_2</zorder>
     <zorder>text_filename_2</zorder>
     <zorder>text_show_2</zorder>
     <zorder>gridFrame_2</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_open"/>
    <addaction name="act_receive"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="act_open">
   <property name="text">
    <string>Send</string>
   </property>
  </action>
  <action name="act_receive">
   <property name="text">
    <string>Receive</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
